Sources/SwiftDocCUtilities/ArgumentParsing/Subcommands/DiffDocCArchive.swift

```diff
@@ -0,0 +1,253 @@
+                    
+                    #### Additions
+                    _New symbols added in \(newerDocCArchiveVersion) that did not previously exist in \(initialDocCArchiveVersion)._
+                                        
+                    \(additionLinks)
+                    
+                    
+                    #### Removals
+                    _Old symbols that existed in \(initialDocCArchiveVersion) that no longer exist in \(newerDocCArchiveVersion)._
+                                        
+                    \(removalLinks)
+                    
+                    """
+            ]
+        }
+        
+        // MARK: - Command Line Options & Arguments
+        
+        @Argument(
+            help: ArgumentHelp(
+                "The version of the initial DocC Archive to be compared.",
+                valueName: "initialDocCArchiveVersion"))
+        var initialDocCArchiveVersion: String
+        
+        @Argument(
+            help: ArgumentHelp(
+                "The path to the initial DocC Archive to be compared.",
+                valueName: "initialDocCArchive"),
+            transform: URL.init(fileURLWithPath:))
+        var initialDocCArchivePath: URL
+        
+        @Argument(
+            help: ArgumentHelp(
+                "The version of the newer DocC Archive to be compared.",
+                valueName: "newerDocCArchiveVersion"))
+        var newerDocCArchiveVersion: String
+        
+        @Argument(
+            help: ArgumentHelp(
+                "The path to the newer DocC Archive to be compared.",
+                valueName: "newerDocCArchive"),
+            transform: URL.init(fileURLWithPath:))
+        var newerDocCArchivePath: URL
+        
+        // MARK: - Execution
+        
+        public mutating func run() throws {
+            let initialDocCArchiveAPIs: [URL] = try findAllSymbolLinks(initialPath: initialDocCArchivePath)
+            let newDocCArchiveAPIs: [URL] = try findAllSymbolLinks(initialPath: newerDocCArchivePath)
+            
+            let initialSet = Set(initialDocCArchiveAPIs.map { $0 })
+            let newSet = Set(newDocCArchiveAPIs.map { $0 })
+            
+            // Compute additions and removals to both sets
+            let additionsToNewSet = newSet.subtracting(initialSet)
+            let removedFromOldSet = initialSet.subtracting(newSet)
+            
+            // The framework name is the path component after "/documentation/".
+            var frameworkName: String = "No_Framework_Name"
+            var potentialFrameworkName = try findFrameworkName(initialPath: initialDocCArchivePath)
+            if potentialFrameworkName == nil {
+                potentialFrameworkName = try findFrameworkName(initialPath: newerDocCArchivePath)
+            }
+            
+            if potentialFrameworkName != nil {
+                frameworkName = potentialFrameworkName ?? "No_Framework_Name"
+            }
```

Comment:
```suggestion
           var frameworkName: String = potentialFrameworkName ?? "No_Framework_Name"
```

The `??`operator unwraps the optional so you don't have to check if it's nil. https://developer.apple.com/documentation/swift/__(_:_:)-8buau